### PR TITLE
feat: [pipe-22241]: add compare pr page

### PR DESCRIPTION
**Pull request by cjlee01**
add compare pr page

image
image

### CODE REVIEW

File: packages/playground/src/components/branch-chooser.tsx
Comment (jessieyu777) [inline packages/playground/src/components/branch-chooser.tsx:1]: May I know what is this prefix prop for the branch selector?

**Review by jessieyu777 — COMMENTED** [review]
a small question about prefix prop in the branch selector

File: packages/playground/src/components/branch-chooser.tsx
Comment (cjlee01) [reply packages/playground/src/components/branch-chooser.tsx:1]: image
in the picture , you see this base: or compare: its just that @jessieyu777

Comment (jessieyu777) [reply packages/playground/src/components/branch-chooser.tsx:1]: I see! all others look good to me

**Review by jessieyu777 — APPROVED** [review]
LGTM

**Pull request merged**